.github/workflows/run_periodic_tests.yml

```diff
@@ -66,54 +61,43 @@ jobs:
           sudo apt install gfortran gcc libopenblas-dev graphviz pandoc
           sudo apt install texlive-full
 
-      # Added fixes to homebrew installs:
-      # rm -f /usr/local/bin/2to3
-      # (see https://github.com/actions/virtual-environments/issues/2322)
       - name: Install MacOS system dependencies
         if: matrix.os == 'macos-latest'
         run: |
-          rm -f /usr/local/bin/2to3*
-          rm -f /usr/local/bin/idle3*
-          rm -f /usr/local/bin/pydoc3*
-          rm -f /usr/local/bin/python3*
+          brew analytics off
           brew update
-          brew install graphviz
-          brew install openblas
+          brew install graphviz openblas gcc gfortran libomp
 
       - name: Install Windows system dependencies
         if: matrix.os == 'windows-latest'
         run: choco install graphviz --version=2.38.0.20190211
 
-      - name: Install standard python dependencies
-        run: |
-          python -m pip install --upgrade pip wheel setuptools nox
-
-      - name: Install SuiteSparse and SUNDIALS on GNU/Linux
-        if: matrix.os == 'ubuntu-latest'
-        run: nox -s pybamm-requires
+      - name: Install SuiteSparse and SUNDIALS on GNU/Linux and macOS
```

Comment:
I enabled this step for macOS for this workflow and other ones (turns out we were not building the IDAKLU component in CI since forever!). We were building wheels without ABI or platform tags (`none-any.whl`) otherwise